tools/apigee-sackmesser/cmd/report/xhybrid/apiproducts.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# <http://www.apache.org/licenses/LICENSE-2.0>
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+urlencode() {
+    echo "\"${*:1}\"" | jq -r '@uri'
+}
+
+echo "<h3>API Products</h3>" >> "$report_html"
+
+mkdir -p "$export_folder/$organization/config/resources/edge/env/$environment/api-products"
+
+sackmesser list "organizations/$organization/apiproducts"| jq -r -c '.[]|.' | while read -r apiProductName; do
+        sackmesser list "organizations/$organization/apiproducts/$(urlencode "$apiProductName")" > "$export_folder/$organization/config/resources/edge/env/$environment/api-products/$(urlencode "$apiProductName")".json
+        elem_count=$(jq '.entries? | length' "$export_folder/$organization/config/resources/edge/env/$environment/api-products/$(urlencode "$apiProductName")".json)
```

Comment:
(for all `elem_count` variables in this PR)

I don't see where the `elem_count` values are used

Reply:
Done

Reply:
@g-devpat Can you do the same for the 8 other `elem_count` variables?

Reply:
@danistrebel Yes, I did that as part of the previous commit for all the helper files